resources/views/ItemList/Components/Filter/ItemFilterPrice.twig

```diff
@@ -0,0 +1,32 @@
+<script type="x/template" id="vue-findologic-item-price">
```

Comment:
Just for my understanding, why didn't we need this for Ceres 4.6? Did Plenty remove this template in 5.0?

Reply:
And was this copied or did you write it yourself?

Reply:
> Just for my understanding, why didn't we need this for Ceres 4.6? Did Plenty remove this template in 5.0?

Plenty now uses `.vue` files. We manually need to override these to be able to use them as before.

> And was this copied or did you write it yourself?

The code was copied straight from plenty. It will be removed once we also switch to `.vue` files.

Reply:
Please create a story with all necessary information to convert our twig files to vue. Also mentioning the files which can be removed like this one. When this is done this discussion can be resolved